samples/server-backend/src/xpoc-common.ts

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+function showError(errMsg: string): void {
+    const errMsgElem = document.getElementById('errorMsg') as HTMLElement;
+    errMsgElem.textContent = errMsg;
+    errMsgElem.style.display = 'block';
+}
+
+function clearError(): void {
+    const errMsgElem = document.getElementById('errorMsg') as HTMLElement;
+    errMsgElem.textContent = '';
+    errMsgElem.style.display = 'none';
+}
+
+function isoToLocalTime(isoTime: string): string {
+    if (!isoTime) {
+        return '';
+    }
+    return new Date(isoTime).toLocaleString();
+}
+
+const prefix = '/fetchManifest?location=';
```

Comment:
It would be good to also copy this file as-is from the client-only sample, and only have the ability to set the prefix (which would be empty by default).